pkg/drivers/none/none.go

```diff
@@ -54,11 +55,13 @@ func NewDriver(hostName, storePath string) *Driver {
 
 // PreCreateCheck checks for correct privileges and dependencies
 func (d *Driver) PreCreateCheck() error {
-	// check that docker is on path
-	_, err := exec.LookPath("docker")
-	if err != nil {
-		return errors.Wrap(err, "docker cannot be found on the path for this machine. "+
-			"A docker installation is a requirement for using the none driver")
+	if d.ContainerRuntime == "" {
```

Comment:
I'd extract this into a method: `isDockerRuntime`

Reply:
Just into a local function ? Sure, that can be done